RecoJets/JetAlgorithms/interface/HEPTopTaggerV2.h

```diff
@@ -0,0 +1,347 @@
+#ifndef __HEPTOPTAGGERV2_HH__
+#define __HEPTOPTAGGERV2_HH__
```

Comment:
please use include guards matching the package name.
(unless there is a good reason not to)